content/security/committers.md

```diff
@@ -137,12 +147,12 @@ employer) at the discretion of the project's security team, providing that
 you make clear that the information is not for public disclosure and that you copy to
 `security@apache.org` or the project's security mailing list any communication regarding the vulnerability.
 
-# CVE names
+<h2 id="ids">CVE IDs<a class="headerlink" href="#ids" title="Permanent link">&para;</a></h2>
 
 [Common Vulnerabilities and Exposures](https://cve.mitre.org/) (CVE)
-IDs are a unique identifiers given to security flaws.  The Apache
+IDs are unique identifiers given to security flaws.  The Apache
```

Comment:
Why do we say 'flaw' here rather than vulnerability?

I've seen hesitancy to call something a vuln until it's determined it is, but in this case it's in the acronym :)

Reply:
changed in the patch